irohad/torii/impl/command_service.cpp

```diff
@@ -57,11 +57,10 @@ namespace torii {
       auto tx_hash = proto_response->transactionHash();
       auto cached_tx_state = cache_->findItem(tx_hash);
       if (cached_tx_state
-          and proto_response->getTransport().tx_status()
-              <= cached_tx_state->tx_status()) {
+          and proto_response->comparePriorities(**cached_tx_state) <= 0) {
```

Comment:
It was a value, which compare function gave, so that `<= 0` meant that first value is less or equal, than the second one, C-style. But as it's going to be replaced by enum class, doesn't matter